tests/scratch_test_helm.py

```
@@ -0,0 +1,5 @@
+import subprocess
+
+
+def test_helm_binary_downloads(helm_binary_path):
+    print(subprocess.run([helm_binary_path, "version"], capture_output=True, text=True).stdout)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)